### PR TITLE
Fix/Using token fee coming from controller

### PR DESCRIPTION
**Pull request by ignaciosantise**
## Summary
* Updated controller to receive token transaction fees

## Screenshots
Screen Shot 2022-09-14 at 11 33 19
Screen Shot 2022-09-14 at 11 33 28

## Github Ticket
https://github.com/Psychedelic/plug-mobile/issues/419

### CODE REVIEW

File: src/screens/flows/Send/components/ReviewSend/index.js [hunk 1]
```diff
@@ -120,14 +117,12 @@ const ReviewSend = ({
...
-              ) : null}
-              <Text
-                type="caption"
-                style={value ? styles.subtitle : styles.title}>
+              <Text type="caption" style={styles.title}>
                 {`${amount?.display} ${token.symbol}`}
               </Text>
+              {value ? (
+                <Text style={styles.subtitle}>${value?.display}</Text>
+              ) : null}
```
Comment (mattgle) [inline src/screens/flows/Send/components/ReviewSend/index.js:125]: Does this work?, or is it a reason why the ternary with null?
```suggestion
              {value && (
                <Text style={styles.subtitle}>${value?.display}</Text>
              )}
```

File: src/screens/flows/Send/components/ReviewSend/index.js [hunk 2]
```diff
@@ -173,10 +168,11 @@ const ReviewSend = ({
           <Row style={styles.row}>
             <Text type="subtitle3">
               {t('reviewSend.totalFee', {
-                value: `${currentFee} ${token?.symbol} ($${Number(
-                  currentUSDFee.toFixed(VISIBLE_DECIMALS)
-                )})`,
+                value: `${token.fee} ${token?.symbol}`,
               })}
+              {feePrice
+                ? ` ($${Number(feePrice?.toFixed(VISIBLE_DECIMALS))})`
```
Comment (mattgle) [inline src/screens/flows/Send/components/ReviewSend/index.js:174]: Have you tried with values that are possible to be fixed and round the number?

**Review by mattgle — COMMENTED** [review]
Just a few comments

File: src/screens/flows/Send/components/ReviewSend/index.js (hunk 1 above)
Comment (ignaciosantise) [reply src/screens/flows/Send/components/ReviewSend/index.js:125]: Sometimes using && causes crashes in renders, mostly when using Text component. So i ended using always the ternary. The last crash i solved was in the transaction item, one specific transaction made the app crash a few months ago.

Comment (mattgle) [reply src/screens/flows/Send/components/ReviewSend/index.js:126]: Great! Didn't know that

**Review by mattgle — APPROVED** [review]

**Pull request merged**